pkg/clamav/scanner.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/fsouza/go-dockerclient"
 	"github.com/openshift/clam-scanner/pkg/clamav"
+	"golang.org/x/net/context"
```

Comment:
@simon3z this is why this PR adds `golang.org/x/net`
@legionus is there a reason we're using this package instead of stdlib `context`?

Reply:
> @legionus is there a reason we're using this package instead of stdlib context?

@ilackarms OK. I will use the `context` from stdlib.